nexus_client_sdk/nexus/algorithms/_remote_algorithm.py

```diff
@@ -86,6 +96,57 @@ async def _run(self, **kwargs) -> list[AlgorithmPayload]:
     def _metric_tags(self) -> dict[str, str]:
         return {"algorithm": self.__class__.alias()}
 
+    @staticmethod
+    def _get_compression_config() -> tuple[dict[str, str], callable]:
+        """
+        Retrieves and validates the compression configuration and function from environment variables.
+        Returns (config dict, compression function).
+        """
+        compression_config_json = os.getenv(RemoteAlgorithm.COMPRESSION_ALGORITHM_ENV)
```

Comment:
This should be moved out to app dependencies, we do not read any config at runtime at this point. Utilise DI.